config/zones/AT.yaml

```diff
@@ -42,6 +42,18 @@ emissionFactors:
       - datetime: '2021-01-01'
         source: electricityMap, 2021 average
         value: 189.1555066532277
+    biomass:
+      datetime: '2021-01-01'
+      source: EU-ETS, ENTSO-E 2021
```

Comment:
Would it make sense to have a `source_link` attribute here? We could then use it to auto-generate a markdown file similar to DATA_SOURCES (or just added to the same), and it would also make it easier to find information :)

Reply:
Or perhaps a top-level object to map the source to the link so we don't have to replicate it for every time the source is used?
Example if DK-DK1 file with sources object:
```yaml
bounding_box:
  - - 10.535889
    - 54.380557
  - - 12.788086
    - 56.36525
capacity:
  battery storage: 0
  biomass: 1015
  coal: 1943
  gas: 1018
  geothermal: 0
  hydro: 7
  hydro storage: 0
  nuclear: 0
  oil: 196
  solar: 1086
  wind: 5165
contributors:
  - corradio
  - tmslaine
sources:
  ENTSO-E 2021:
    link: <link>
    doi: <doi number>
...
```
This way we don't create duplicates for each production mode for each year it's used (if it's reused).

Reply:
Great idea @VIKTORVAV99! Just integrated the change, and I'll follow up with a script to automatically update data sources